### PR TITLE
Update Mac CLI Installation Instructions

### PR DESCRIPTION
**Pull request by kwcanuck**
This fix resulted from this [Slack conversation](https://astronomer.slack.com/archives/C03HRT0MRPD/p1674683875620649).

### CODE REVIEW

**Comment by kwcanuck** [comment]
@kushalmalani could you review the changes and let me know if they can be implemented? Thanks!

**Review by kushalmalani — APPROVED** [review]
LGTM. The CLI release notes link is broken. gives me a 404 not found

**Comment by kwcanuck** [comment]
Thanks for the quick review and feedback @kushalmalani. The link issue has been resolved.

**Comment by paolaperaza** [comment]
Hey @kwcanuck -- I did a couple things here.

- Created a new section at the same level as `#### installation` for users to run `astro version` and verify that the thing was installed properly
- Put the `brew update` command into a tip. I actually didn't realize this was already in our docs, and I agree that the current structure isn't great. I think the tip is fine -- it really is if you run into an issue. I can see us having a dedicated "troubleshooting" section for CLI installs if we get more feedback, but for now let's keep it there
- I removed the `astrocloud` steps. Honestly at this rate -- we have so many Astro customers and there's no way more than 1-2% have `astrocloud`, if any. Let's remove it and our team can give guidance if asked

**Comment by kwcanuck** [comment]
@paolaperaza unfortunately, your changes don't work. Resolving the issue is a two-step process and that's a procedure. Procedures don't belong in tips. Also, we're already overusing tips and other annotations. I think my previous changes worked better as it avoided multiple headings and a tip annotation.

I've modified the documentation again. Please review.

File: astro/cli/install-cli.md [hunk 1]
```diff
@@ -43,33 +43,32 @@ To install the latest version of the Astro CLI, run the following command:
...
 
-    ```sh
-    brew install astro
-    ```
+To verify that the correct Astro CLI version was installed, run:
 
-3. Run the following command to verify the correct Astro CLI version was installed:
+```sh
+astro version
+```
```
Comment (jwitz) [inline astro/cli/install-cli.md:60]: Add space between bottom of code fence and `#### Resolve installation issues`

**Review by jwitz — APPROVED** [review]
Big 🥂 to finally removing the astrocloud migration instructions!

File: astro/cli/install-cli.md [hunk 2]
```diff
@@ -43,33 +43,32 @@ To install the latest version of the Astro CLI, run the following command:
...
+#### Resolve installation issues
```
Comment (jwitz) [inline astro/cli/install-cli.md:61]: My only concern is that this process only solves one specific issue, as opposed to all issues that could arise. 

Would it also make sense to rename this "Troubleshoot installation issues"? I feel like that's the more common term people would search for

Comment (kwcanuck) [reply astro/cli/install-cli.md:61]: @jwitz I considered using _troubleshooting_ or _troubleshoot_, but selected _resolve_ as it's the appropriate choice here. _Troubleshoot_ or _troubleshooting_ implies tips for investigating and then resolving an issue. That isn't the case here.

Here is the definition of resolve:

_settle or find a solution to (a problem, dispute, or contentious matter)_

Here is the definition of troubleshoot:

_solve serious problems for a company or other organization_

_Resolve_ is the better choice as it is more accurate term here. It's the provision of a _resolution_ to two common issues. It's not a serious problem.

This section is visible from the installation section, so it's unlikely that users will need to search for it using troubleshoot. Also, we would need to expand the title to _Troubleshoot Astro CLI installation issues_ to make the intent of the information clear when searching. This is a simple, straightforward change and it works as intended. There is no need to complicate the structure or titles.

File: astro/cli/install-cli.md (hunk 1 above)
Comment (kwcanuck) [reply astro/cli/install-cli.md:60]: Done.

File: astro/cli/install-cli.md [hunk 3]
```diff
@@ -147,13 +147,12 @@ To install a specific version of the CLI, specify the version number as a flag a
 
 <pre><code parentName="pre">{`curl -sSL install.astronomer.io | sudo bash -s -- v${siteVariables.cliVersion}`}</code></pre>
 
-If you specify only a major version, this command installs the latest minor or patch version available for the major version. If you specify only a major version, this command installs the latest minor or patch version available for the major version. For a list of all available versions, see the [CLI release notes](cli/release-notes.md).
+If you specify only a major version, this command installs the latest minor or patch version available for the major version. If you specify only a major version, this command installs the latest minor or patch version available for the major version. For a list of all available versions, see the [CLI release notes](/astro/cli/release-notes.md).
```
Comment (jwitz) [inline astro/cli/install-cli.md:150]:
```suggestion
If you specify only a major version, this command installs the latest minor or patch version available for the major version. If you specify only a major version, this command installs the latest minor or patch version available for the major version. For a list of all available versions, see the [CLI release notes](cli/release-notes.md).
```

File: astro/cli/install-cli.md [hunk 4]
```diff
@@ -43,33 +43,33 @@ To install the latest version of the Astro CLI, run the following command:
...
-brew install astro@<major.minor.patch-version>
-```
+1. Run the following command to update Homebrew and all package definitions (formulae):
+
+    ```sh
+    brew update
+    ```
 
-If you specify only a major version, this command installs the latest minor or patch version available for the major version. For a list of all available versions, see the [CLI release notes](cli/release-notes.md). If you specify only a major version, this command installs the latest minor or patch version available for the major version. For a list of all available versions, see the [CLI release notes](cli/release-notes.md).
+2. Install the Astro CLI. See [Installation](astro/cli/install-cli.md?tab=mac#installation).
```
Comment (jwitz) [inline astro/cli/install-cli.md:72]:
```suggestion
2. Install the Astro CLI. See [Installation](cli/install-cli.md?tab=mac#installation). 
```

**Review by paolaperaza — APPROVED** [review]

**Pull request merged**